architecture-decision-records/0025-automatic-code-merge.md

```diff
@@ -0,0 +1,28 @@
+#25. Automatic code merging
+
+**Date:** 2022-03-15
+
+## Context
+We have two services now, dependabot and Scala Steward, which create pull requests for dependency updates. 
+We are getting around 5 or 6 of these per day which need to be merged in turn and can take a lot of time. We looked at using an automatic merging service.
```

Comment:
Maybe reword this slightly: _... an automatic merging service to reduce the amount of time needed on merging dependency updates._